redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -199,10 +200,6 @@ internal open class UIViewLazyList() : LazyList<UIView>, ChangeListener {
       override fun scrollViewDidEndScrollingAnimation(scrollView: UIScrollView) {
         isDoingProgrammaticScroll = false
       }
-
-      override fun scrollViewDidScrollToTop(scrollView: UIScrollView) {
```

Comment:
This hook is called well in advance of `scrollViewDidEndScrollingAnimation`; we want to continue ignoring `scrollViewDidScroll` events until the scroll is completed, now that the `scrollProcessor` already knows "where the puck is going" (to quote @swankjesse).